### ISSUE TITLE
Metadata not being reported following new Accessibility 1.1 Recommendations

### ISSUE THREAD

**Issue by clapierre** [daisy/ace-gui#163]
When including the new refines attribute linking the certifier to the conformsTo and the certifersCredentials to the certifiedby this metadata is not reported.

Following [example 7 in the Accessibility 1.1 Specification](https://w3c.github.io/epub-specs/epub33/a11y/#example-expressing-a-basic-credential) Only the ConformsTo is reported

This is using the latest 1.1.5 

If you remove the refines statements, then the metadata is reported.

in contents.opf 
```
   <link id="conformance" rel="dcterms:conformsTo" href="http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa/"/>
    <meta refines="#conformance" id="certifier" property="a11y:certifiedBy">Benetech</meta>
    <meta refines="#certifier" property="a11y:certifierCredential">https://bornaccessible.org/certification/gca-credential/</meta>

```
Only conformsTo shows in the metadata (ie. since it does not refine another metadata property.)

**Comment by clapierre** [comment]
Removing the refines statement in both cases results in all 3 pieces of metadata being presented in the report.

**Comment by danielweck** [comment]
This was fixed in DAISY Ace App v1.3.2
https://github.com/daisy/ace-gui/releases/tag/v1.3.2

**Comment by danielweck** [comment]
Related issue: https://github.com/daisy/ace/issues/383
